launchable/test_runners/go_test.py

```diff
@@ -71,5 +72,14 @@ def path_builder(case: TestCase, suite: TestSuite, report_file: str) -> TestPath
     client.run()
 
 
+def format_same_bin(s: str) -> Dict[str, str]:
```

Comment:
This is to support test profile that go-test are identified with `class` and `testcase`.